support-workers/src/main/scala/com/gu/support/workers/exceptions/RetryImplicits.scala

```diff
@@ -28,6 +28,12 @@ object RetryImplicits {
         // response from Zuora during maintenance windows
         new RetryLimited(message = e.getMessage, cause = throwable)
 
+      case e @ (_: WebServiceClientError) if e.codeBody.code == "429" =>
+        // We are retrying on 429 (rate limit) errors now because we have been hitting Zuora's rate limit at
```

Comment:
is this code zuora specific? I notice also 401 retry seems to be for zuora errors in particular, which is probably ok as it will fail after a short delay just wanted to make sure that's intended.

Reply:
It's not Zuora specific no because the error we get is an http error however I think it makes sense to retry any rate limit error.